src/qml/WiFiPage.qml

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2023 - Arseniy Movshev <dodoradio@outlook.com>
+ *               2017-2022 - Chupligin Sergey <neochapay@gmail.com>
+ *               2021 - Darrel Griët <idanlcontact@gmail.com>
+ *               2016 - Sylvia van Os <iamsylvie@openmailbox.org>
+ *               2015 - Florent Revest <revestflo@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* A large proportion of this code has been referenced from Nemomobile-UX Glacier-Settings, published at https://github.com/nemomobile-ux/glacier-settings/blob/master/src/plugins/wifi/WifiSettings.qml
+ */
+
+import QtQuick 2.9
+import org.asteroid.controls 1.0
+import org.asteroid.utils 1.0
+import Connman 0.2
+import QtQuick.VirtualKeyboard 2.4
+
+Item {
+    id: root
+    TechnologyModel {
+        id: wifiModel
+        name: "wifi"
+        onCountChanged: {
+            console.log("COUNT CHANGE " + count)
```

Comment:
I know this is a draft but since I'm at it, let me think out loud. I believe we should try to keep the console.logs that make it to production ~minimal and "to the point". There's a sweet spot between not providing any debug info in our logs (basically the current situation) and drowning useful information in tons of logs (what would happen if we start to do this sort of things)